scripts/reboot_helper.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
```

Comment:
Not sure if `reboot_helper.py` script is not copied to the target image on build

Reply:
Completed. Thank you!